src/restconfig/src/test/java/org/geoserver/catalog/rest/CoverageStoreTest.java

```diff
@@ -373,8 +392,8 @@ public void testDeletePurgeAllAfterConfigure() throws Exception {
 
     @Test
     public void testPut2() throws Exception {
-        Document dom = getAsDOM( "/rest/workspaces/wcs/coveragestores/BlueMarble.xml");
-        assertXpathEvaluatesTo("GeoTIFF", "/coverageStore/type", dom );
+        //Document dom = getAsDOM( "/rest/workspaces/wcs/coveragestores/BlueMarble.xml");
+        //assertXpathEvaluatesTo("GeoTIFF", "/coverageStore/type", dom );
```

Comment:
That was actually just something I forgot to uncomment after debugging, thanks for catching it.